Mathlib/MeasureTheory/Measure/ProbabilityMeasure.lean

```diff
@@ -548,4 +561,42 @@ end ProbabilityMeasure -- namespace
 
 end map -- section
 
+section MonoidalProduct
+
+open Measure
+
+/-- The monoidal product is a measurable function from the product of probability spaces over
+``α`` and ``β`` into the type of probability spaces over ``α × β``. Lemma 4.1 of
+https://doi.org/10.1016/j.aim.2020.107239.
+-/
+theorem ProbabilityMeasure.measurable_prod {α β : Type*} [MeasurableSpace α] [MeasurableSpace β] :
+    Measurable (fun (μ : ProbabilityMeasure α × ProbabilityMeasure β)
+      ↦ μ.1.toMeasure.prod μ.2.toMeasure) := by
+  apply Measurable.measure_of_isPiSystem generateFrom_prod.symm isPiSystem_prod _ (by simp)
```

Comment:
There is a specialized corollary for probability measures that avoids the `(by simp)`:
```suggestion
  apply Measurable.measure_of_isPiSystem_of_isProbabilityMeasure generateFrom_prod.symm
    isPiSystem_prod _
```